src/Command/GenerateSchemaCommand.php

```diff
@@ -57,98 +83,91 @@ protected function execute(InputInterface $input, OutputInterface $output)
                             if ($rowNumber === 0) {
                                 return;
                             }
-                            
+
                             if (0 === strpos($rowNode->filter('td:nth-child(3)')->text(), 'Optional')) {
                                 $required = false;
                             } else {
                                 $required = true;
                             }
-                            
+
                             $fields[] = [
-                                'name' => $rowNode->filter('td:nth-child(1)')->text(),
-                                'roughType' => $rowNode->filter('td:nth-child(2)')->text(),
+                                'name'        => $rowNode->filter('td:nth-child(1)')->text(),
+                                'roughType'   => $rowNode->filter('td:nth-child(2)')->text(),
                                 'description' => $rowNode->filter('td:nth-child(3)')->text(),
-                                'required' => $required,
+                                'required'    => $required,
                             ];
                         });
 
-                        $this->schema['objects'][] = [
-                            'name' => $methodOrObjectName,
-                            'link' => self::URL.$node->filter('a')->attr('href'),
-                            'fields' => $fields
+                        $this->schema['types'][$methodOrObjectName] = [
+                            'name'         => $methodOrObjectName,
+                            'link'         => $node->filter('a')->attr('href'),
```

Comment:
```suggestion
                            'link'         => self::URL.$node->filter('a')->attr('href'),
```
The generated output was missing the full URL

Reply:
i got bot api html from https://core.telegram.org/bots/api
1. Open in browser
2. Save to schema.html
3. Parse
and i got full url

> `/**
 * https://core.telegram.org/bots/api#chat
 *
 * This object represents a chat. 
 */`